Source/cmake/OptionsWinCairo.cmake

```diff
@@ -67,6 +67,13 @@ SET_AND_EXPOSE_TO_BUILD(ENABLE_DEVELOPER_MODE ${DEVELOPER_MODE})
 
 SET_AND_EXPOSE_TO_BUILD(HAVE_OS_DARK_MODE_SUPPORT 1)
 
+# See if OpenSSL implementation is BoringSSL
+cmake_push_check_state()
+set(CMAKE_REQUIRED_INCLUDES "${OPENSSL_INCLUDE_DIR}")
+set(CMAKE_REQUIRED_LIBRARIES "${OPENSSL_LIBRARIES}")
+WEBKIT_CHECK_HAVE_SYMBOL(USE_BORINGSSL OPENSSL_IS_BORINGSSL openssl/ssl.h)
```

Comment:
If BoringSSL has `OPENSSL_IS_BORINGSSL` macro, do we have to define `USE_BORINGSSL`? How about an idea using `defined(OPENSSL_IS_BORINGSSL)`?

Reply:
Combing with the question above yes some algorithm's are not present in BoringSSL. The one that is going to have to be gated on whether or not its BoringSSL is `CryptoAlgorithmAES_CFB`.

With regard to using `OPENSSL_IS_BORINGSSL` directly it was more to fit with the conventions in WebKit and to allow us to control the value inside the build rather than relying on the outside one.

Reply:
You can see what algorithms are present by looking at https://diafygi.github.io/webcrypto-examples/

It looks like AES-CFB is actually no longer in the spec